Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM daskdev/dask:latest
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    make \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /opt/data_apps
```

Comment:
My preference would be `/usr/local/src` as this is a more common source location.

Reply:
> This looks like a great change and should make our testing environment more consistent.
Thank you!

>     * Switching to kind introduces Docker as a test dependency. Minikube does not have the same requirement. This should be noted somewhere.

My understanding is that `minikube start --vm-driver=none` implicitly uses host's docker - it comes pre-installed on CircleCI machine

- [ ] Though I have left the `minikube` make commands if you prefer to switch back?

>     * You've removed the CI conda environment in favour of the `daskdev/dask:latest` image. These are not equivalent. Specifically the CI environment installs `dask` and `distributed` from master, which allows `dask-kubernetes` to be tested against unreleased versions and for the libraries to vbe developed in parallel. Therefore `requirements-dev.txt` should probably include those.

- Thanks, I will make sure to update it accordingly
  - [ ] add dask@master and distributed@master to `requirements-dev.txt`

Reply:
:+1: